app/facades/garden_facade.rb

```diff
@@ -1,4 +1,10 @@
 class GardenFacade
+  def self.create_garden_objects(users_gardens) 
+    users_gardens.map do |garden|
+      response = garden_details(garden)
+    end
+  end
```

Comment:
Variable response isn't being used. Not a huge problem.

Reply:
Removed!